src/stream/src/executor/sort_buffer.rs

```diff
@@ -227,3 +231,32 @@ impl<S: StateStore> SortBuffer<S> {
         Ok(())
     }
 }
+
+fn fix_row<S: StateStore>(
```

Comment:
Could you please shed some light on it?

Reply:
Renamed the function and added some comment.